app/src/androidTest/java/com/github/sdpcoachme/messaging/ChatActivityTest.kt

```diff
@@ -441,6 +441,23 @@ class ChatActivityTest {
         }
     }
 
+    // simulates push notifications where the user logs out after receiving the notification and then logs in
```

Comment:
Very good to have that comment for the understanding. Otherwise, it would be a bit hard to completely understand what behaviour is tested here.